### PR TITLE
Log a new workout with title

### PR DESCRIPTION
**Pull request by tsrandrei**
### Description

Created model Workout, with relative update of the routes.
Added controller with views.
Created rspecs, with specs passing.

The model have a title, with validation on title field, as required by AC.

Accessing 0.0.0.0:3000/workouts/new it's possible to insert a new workout with title. 

### Checklist

Make sure that all steps a checked before the merge

- [x] RSpec tests are passing
- [x] Tested manually

### CODE REVIEW

File: app/views/workouts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<%= form_with scope: :workout, url: workouts_path, local: true do |f| %>
```
Comment (lshepstone) [inline app/views/workouts/new.html.erb]: Why is `f` use here instead of `form`?

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
 Rails.application.routes.draw do
   devise_for :users
 
+  resources :workouts
```
Comment (lshepstone) [inline config/routes.rb]: How come you're allowing all actions on this resource when the controller supports only a few?

File: db/migrate/20190208121005_create_workouts.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateWorkouts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :workouts do |t|
+      t.string :title
```
Comment (lshepstone) [inline db/migrate/20190208121005_create_workouts.rb]: `title` is required at the model level, so why is it not `null: false` here?

File: spec/forms/workout_form_spec.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Workout, type: :model do
```
Comment (lshepstone) [inline spec/forms/workout_form_spec.rb]: This spec is for a class that doesn't exist?

File: app/controllers/workouts_controller.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class WorkoutsController < ApplicationController
+  def index
+    @workouts = Workout.all
```
[the remark this answers is not in the document]
Comment (tsrandrei) [reply app/controllers/workouts_controller.rb]: #15 ?

File: app/views/workouts/new.html.erb (hunk 1 above)
Comment (tsrandrei) [reply app/views/workouts/new.html.erb]: fixed in #16

File: config/routes.rb (hunk 2 above)
Comment (tsrandrei) [reply config/routes.rb]: fixed in #16

File: db/migrate/20190208121005_create_workouts.rb (hunk 3 above)
Comment (tsrandrei) [reply db/migrate/20190208121005_create_workouts.rb]: fixed in #16

File: spec/forms/workout_form_spec.rb (hunk 4 above)
Comment (tsrandrei) [reply spec/forms/workout_form_spec.rb]: fixed in #16

File: app/views/workouts/new.html.erb (hunk 1 above)
Comment (tsrandrei) [reply app/views/workouts/new.html.erb]: This is fixed in c603eeff

**Pull request merged**